sdks/python/apache_beam/runners/portability/flink_runner_test.py

```diff
@@ -262,6 +262,12 @@ def test_sdf_with_sdf_initiated_checkpointing(self):
     def test_sdf_synthetic_source(self):
       raise unittest.SkipTest("BEAM-2939")
 
+    def test_sdf_wrapper_overrides_read(self):
```

Comment:
Perhaps these tests should simply be moved out of this class (which has many subclasses) rather than skipped here.

Reply:
If flink supports SDF and want to use SdfBoundedSourceWrapper, flink runner is expected to replace Read transforms with SdfBoundedSourceWrapper.